cca_utils/ldap3_utils.py

```diff
@@ -214,14 +211,26 @@ def ldap_remove_members_from_group(groupcn, remove_members):
 
     groupdn = "cn={groupcn},{ou}".format(groupcn=groupcn, ou=settings.LDAP_GROUPS_OU)
     mod_attrs = {}
-    if len(remove_members) > 0:
-        mod_attrs['memberUid'] = [MODIFY_DELETE, remove_members]
 
+    # Make sure remove_members is actually a list
+    if isinstance(remove_members, list):
+        del_members = []
+        # Remove any non-existent LDAP users from list
+        for person in remove_members:
+            del_members.append('uid={u},{ou}'.format(ou=settings.LDAP_PEOPLE_OU, u=person))
+            if not ldap_get_user_data(person):
+                del_members.remove(person)
+            print(person)
```

Comment:
Please remove these print statements. They'll crash the server if we ever were to switch from gunicorn to apache (you can use the logger if you need to capture these).